src/terminal/input/terminalInput.hpp

```diff
@@ -69,14 +70,28 @@ namespace Microsoft::Console::VirtualTerminal
         std::optional<wchar_t> _leadingSurrogate;
 
         std::optional<WORD> _lastVirtualKeyCode;
+        DWORD _lastControlKeyState = 0;
+        uint64_t _lastLeftCtrlTime = 0;
+        uint64_t _lastRightAltTime = 0;
+        std::unordered_map<int, std::wstring> _keyMap;
+        std::wstring _focusInSequence;
+        std::wstring _focusOutSequence;
 
         til::enumset<Mode> _inputMode{ Mode::Ansi, Mode::AutoRepeat };
         bool _forceDisableWin32InputMode{ false };
 
-        [[nodiscard]] OutputType _makeCharOutput(wchar_t ch);
-        [[nodiscard]] static OutputType _makeEscapedOutput(wchar_t wch);
-        [[nodiscard]] static OutputType _makeWin32Output(const KEY_EVENT_RECORD& key);
-        [[nodiscard]] static OutputType _searchWithModifier(const KEY_EVENT_RECORD& keyEvent);
+        // In the future, if we add support for "8-bit" input mode, these prefixes
+        // will sometimes be replaced with equivalent C1 control characters.
+        static constexpr auto CSI = L"\x1B[";
+        static constexpr auto SS3 = L"\x1BO";

```

Comment:
Would it make sense to already name these two with `_member` names so that we need to modify less line in the future if we were to make such a change?